Code/Framework/AzAutoGen/CMakeLists.txt

```diff
@@ -11,4 +11,7 @@ ly_add_target(
     NAMESPACE AZ
     FILES_CMAKE
         azautogen_files.cmake
+    INCLUDE_DIRECTORIES
+        INTERFACE
+            .
```

Comment:
I was thinking we just move Code/Framework/AzAutoGen -> scrtips/AzAutoGen and update the LyAutoGen.cmake script to point there

Reply:
This target isn't actually a real project.
It is just used to add the AzAutoGen.py script to the Visual Studio solution.

That is the only file in `azautogen_files.cmake`

Reply:
I was thinking this can solve other py cases... if some module wanted to "export" some py files, they can just drop it into the "public folder". We can also just move `AzAutoGen.py` to `cmake` since thats where it is used and is somehow part of the build..

Reply:
I think the cmake directory is an appropriate place for that script.